build/dev-fast.js

```diff
@@ -20,21 +20,20 @@
 
 const path = require('path');
 const {build} = require('esbuild');
-const commander = require('commander');
 const outFilePath = path.resolve(__dirname, '../dist/echarts.js');
 
 const umdWrapperHead = `(function (root, factory) {
     if (typeof define === 'function' && define.amd) {
         // AMD. Register as an anonymous module.
         define(['exports'], factory);
-    } else if (typeof exports === 'object' && typeof exports.nodeName !== 'string') {
+    } else if (typeof module === 'object' && typeof module.nodeName !== 'string') {
         // CommonJS
-        factory(exports);
```

Comment:
Change `exports` to `module` may behave differently in the CommonJS environment.

Reply:
As for the UMD header, it may be better to use the same block with rollup. 
```js
(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
    typeof define === 'function' && define.amd ? define(['exports'], factory) :
    (global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory(global.echarts = {}));
}(this, (function (exports) {
    // ...
})));
```

Reply:
The reason why I have change `exports` to `module` is that `esbuild` generate `module.exports = ___toCommonJS(****)` at the end of file if we chose `cjs` format.
Cause in browser environment, `module` is not a global variable as in `node`.  It will throw `module is not defined` in browser.

Reply:
JS spec is not my area of expertise. To be honestly, I'm still confused by [esbuild](https://github.com/evanw/esbuild/pull/1849)